examples/low_frequency/magnetic/2d-axi_magnetostatic_actuator.py

```diff
@@ -0,0 +1,187 @@
+# # 2D Axi-symmetric Actuator - Force Calculation
+#
+# This example demonstrates how to leverage both: axi-symmetry and the magnetostatic solver
+# to calculate the forces experienced by the anchor of an actuator due to change in
+# current and anchor location
+#
+# Most examples can be described as a series of steps that comprise a workflow.
+# 1. Import packages and instantiate the application.
+# 2. Do something useful and interesting like creating the model, assign materials and boundary conditions, etc.
+# 3. Run one or more analyses.
+# 4. View the results.
+#
+# Keywords: **Maxwell2D**, **axi-symmetry** **magnetostatic**, **translational motion**, **installation example**
+
+# ## Prerequisites
+#
+# ### Perform imports
+
+# +
+import os
+import tempfile
+import time
+
+import ansys.aedt.core  # Interface to Ansys Electronics Desktop
+
+# -
+
+# ### Define constants
+# Constants help ensure consistency and avoid repetition throughout the example.
+
+AEDT_VERSION = "2025.1"
+NUM_CORES = 4
+NG_MODE = False  # Open AEDT UI when it is launched.
+
+
+# ### Create temporary directory
+#
+# Create a temporary working directory.
+# The name of the working folder is stored in ``temp_folder.name``.
+#
+# > **Note:** The final cell in the notebook cleans up the temporary folder. If you want to
+# > retrieve the AEDT project and data, do so before executing the final cell in the notebook.
+
+temp_folder = tempfile.TemporaryDirectory(suffix=".ansys")
+
+# ### Launch Maxwell 2d
+#
+# Create an instance of
+# the ``Maxwell2d`` class. The Ansys Electronics Desktop will be launched
+# with an active Maxwell2D design. The ``m2d`` object is subsequently
+# used to
+# create and simulate the actuator model.
+
+project_name = os.path.join(temp_folder.name, "2d_axi_magsta_actuator.aedt")
+m2d = ansys.aedt.core.Maxwell2d(
+    project=project_name,
+    solution_type="MagnetostaticZ",
+    version=AEDT_VERSION,
+    non_graphical=NG_MODE,
+    new_desktop=True,
+)
+
+# ## Model Preparation
+#
+# ### Define actuator's housing point location
```

Comment:
@jvela018 please check the headers level everywhere in the code. They don't seem to be consistent. You sometimes use different levels in different sections. Is there a reason? 
Try to open it with Jupytext and see if they are consistent.